recipes/rpbasicdesign/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "rpbasicdesign" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "2c3123bbb2daabba3f7e21ca6896a2bf851b1c2ce1dbb446ab707a198a9853c0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
```

These options are included by default these days (via environment variables in conda-build):
```suggestion
  script: {{ PYTHON }} -m pip install -vv .
```